pages/portals/ood.md

```diff
@@ -126,6 +126,9 @@ When the Jupyter app starts, it will launch these additional modules with it and
 **Loading additional software modules with VSCode:**  
 This works the same way as described above for the Jupyter Lab/Notebook app.  Any software modules listed in this box will be loaded when the VSCode session starts.  
 
+There are a few oddities to note about how VSCode functions at CCR:
```

Comment:
This section is still not showing up as a list - they're all in a single paragraph